pipelinewise/cli/pipelinewise.py

```diff
@@ -1729,6 +1731,37 @@ def _run_post_import_tap_checks(
 
         return errors
 
+    def _cleanup_tap_state_file(self) -> None:
+        tables = self.args.tables
+        state_file = self.tap['files']['state']
+        if tables:
+            self._clean_tables_from_bookmarks_in_state_file(state_file, tables)
+        else:
+            try:
+                os.remove(state_file)
+            except FileNotFoundError:
+                pass
```

Comment:
We have a helper function in `utils.py` called `silentremove` which you can use to remove files